flow/workflows/cdc_flow.go

```diff
@@ -482,12 +431,17 @@ func addCdcPropertiesSignalListener(
 
 func CDCFlowWorkflow(
 	ctx workflow.Context,
-	cfg *protos.FlowConnectionConfigs,
+	flowJobName string,
+	// cfg *protos.FlowConnectionConfigs,
```

Comment:
this breaks backwards compatibility. Should still take config & pull name from that to maintain function signature

Reply:
no, only pass cfg. new code should pass an empty cfg with only FlowJobName present. Code will fill `cfg` from catalog in that scenario